tests/installation/yast2_firstboot.pm

```diff
@@ -73,12 +74,6 @@ sub firstboot_root {
 }
 
 sub firstboot_hostname {
-    if (check_screen('bsc1173298', 30)) {
```

Comment:
Could you please move this change to separate commit, as we remove workaround here.

Reply:
done